R/S3_summary.R

```diff
@@ -57,23 +57,31 @@ summary.iv_robust <- function(object, ...) {
 
 
 summary_lm_model <- function(object) {
-  return_list <-
-    object[c(
-      "call",
-      "k",
-      "rank",
-      "df.residual",
-      "r.squared",
-      "adj.r.squared",
-      "fstatistic",
+
+  out_values <- c(
+    "call",
+    "k",
+    "rank",
+    "df.residual",
+    "res_var",
+    "weighted",
+    "se_type",
+    "fes",
+    "r.squared",
+    "adj.r.squared",
+    "fstatistic"
+  )
+  # Different returns if fixed effects in the output
+  if (object[["fes"]]) {
+    out_values <- c(
+      out_values,
       "proj_r.squared",
       "proj_adj.r.squared",
-      "proj_fstatistic",
-      "res_var",
-      "weighted",
-      "se_type",
-      "fes"
-    )]
+      "proj_fstatistic"
```

Comment:
was the issue that you were getting nulls in return list when no fe's in the model?

Reply:
Yes